test/e2e/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/onsi/ginkgo v1.10.3
 	github.com/onsi/gomega v1.7.1
 	k8s.io/api v0.0.0-20191121015604-11707872ac1c
-	k8s.io/apimachinery v0.0.0-20191121015412-41065c7a8c2a
+	k8s.io/apimachinery v0.17.0
 	k8s.io/client-go v11.0.0+incompatible
```

Comment:
It is generally not a good idea to use different release of k8s.io libraries together.